torch/csrc/jit/passes/guard_elimination.cpp

```diff
@@ -271,9 +272,26 @@ struct GuardElimination {
     case aten::reciprocal:
     case aten::addcmul:
     case aten::where:
+    case aten::bitwise_not:
+    case aten::bitwise_and:
+    case aten::bitwise_or:
+    case aten::bitwise_xor:
+    case aten::_cast_Float:
+    case aten::_cast_Long:
       return checkInputs(n, no_exceptions, true);
+    case aten::softmax:
+      return checkInputs(n, std::unordered_set<size_t>{1}, true);
+    case aten::multinomial:
+      return checkInputs(n, std::unordered_set<size_t>{2, 3}, false);
+    case aten::flatten:
+    case aten::argmax:
+    case aten::squeeze:
     case aten::avg_pool2d:
-    return checkInputs(n, no_exceptions, false);
+      return checkInputs(n, no_exceptions, false);
+    case aten::conv1d:
+    case aten::conv2d:
+    case aten::conv3d:
+      return checkInputs(n, std::unordered_set<size_t>{2, 6}, false);
```

Comment:
the 6th argument of convXd seems to be the number of groups? Shouldn't that affect output shapes even if input and weights' shapes are fixed?

Reply:
Groups affects the result values but not the shape.